### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by shomurali**
first draft of lecture 1

### CODE REVIEW

**Comment by andrewpbray** [comment]
hey thanks, I'll start taking a look right now!

File: lec1.qmd [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: "lec1"
+author: "stat 20 staff"
+format: html
+echo: false
+---
+
+## Quarto
```
Comment (andrewpbray) [inline lec1.qmd:8]: why did you use this title?

Comment (shomurali) [reply lec1.qmd:8]: It seemed apt.

Comment (andrewpbray) [inline lec1.qmd:8]: Why did you do this?

Comment (shomurali) [reply lec1.qmd:8]: It seemed appropriate.

**Review by andrewpbray — APPROVED** [review]
looks good to me!

**Pull request merged**